### ISSUE TITLE
预测结果全是 "{{{{{{{{{{{{{{{{{{{{{{"

### ISSUE THREAD

**Issue by 1667** [LinXueyuanStdio/LaTeX_OCR_PRO#26]
2020-07-27 14:30:36,244:INFO: Epoch 334/430
2020-07-27 14:30:39,214:INFO: - Training:  - 2s - loss: 45.024583 - perplexity: 66662036885159257554076190113792.000000 - lr: 0.000242
2020-07-27 14:30:39,214:INFO: - Config: (before evaluate, we need to see config)
2020-07-27 14:30:39,215:INFO: {
    "export_name": "data.json",
    "dir_images_train": "data/small/images/train/",
    "dir_images_test": "data/small/images/test/",
    "dir_images_val": "data/small/images/val/",
    "path_matching_train": "data/small/matching/train.matching.txt",
    "path_matching_val": "data/small/matching/test.matching.txt",
    "path_matching_test": "data/small/matching/val.matching.txt",
    "path_formulas_train": "data/small/formulas/train.formulas.norm.txt",
    "path_formulas_test": "data/small/formulas/test.formulas.norm.txt",
    "path_formulas_val": "data/small/formulas/val.formulas.norm.txt",
    "max_iter": null,
    "max_length_formula": 150,
    "bucket_train": true,
    "bucket_val": true,
    "bucket_test": true,
    "buckets": [
        [
            240,
            100
        ],
        [
            320,
            80
        ],
        [
            400,
            80
        ],
        [
            400,
            100
        ],
        [
            480,
            80
        ],
        [
            480,
            100
        ],
        [
            560,
            80
        ],
        [
            560,
            100
        ],
        [
            640,
            80
        ],
        [
            640,
            100
        ],
        [
            720,
            80
        ],
        [
            720,
            100
        ],
        [
            720,
            120
        ],
        [
            720,
            200
        ],
        [
            800,
            100
        ],
        [
            800,
            320
        ],
        [
            1000,
            200
        ],
        [
            1000,
            400
        ],
        [
            1200,
            200
        ],
        [
            1600,
            200
        ],
        [
            1600,
            1600
        ]
    ]
}
2020-07-27 14:30:39,216:INFO: {
    "export_name": "vocab.json",
    "unk": "_UNK",
    "pad": "_PAD",
    "end": "_END",
    "path_vocab": "data/small/formulas/vocab.txt",
    "min_count_tok": 0
}
2020-07-27 14:30:39,216:INFO: {
    "export_name": "training.json",
    "device": "cuda:0",
    "criterion_method": "CrossEntropyLoss",
    "n_epochs": 430,
    "batch_size": 2,
    "dropout": 127,
    "clip": -1,
    "lr_method": "Adam",
    "lr_scheduler": "CosineAnnealingLR",
    "lr_init": 0.1,
    "lr_min": 0.0001,
    "start_decay": 6,
    "end_decay": 13,
    "lr_warm": 0.001,
    "end_warm": 0
}
2020-07-27 14:30:39,216:INFO: {
    "export_name": "model.json",
    "model_name": "Img2Seq",
    "encoder_cnn": "vanilla",
    "positional_embeddings": true,
    "attn_cell_config": {
        "cell_type": "lstm",
        "num_units": 512,
        "dim_e": 256,
        "dim_o": 512,
        "dim_embeddings": 80
    },
    "decoding": "beam_search",
    "beam_size": 2,
    "div_gamma": 1,
    "div_prob": 0,
    "max_length_formula": 160
}
2020-07-27 14:30:39,217:INFO: - Evaluating...
2020-07-27 14:30:40,705:INFO: - Eval:  BLEU-4 is 0.57  ||  ExactMatchScore is 0.00  ||  EditDistance is 6.79  ||  perplexity is -197160514126517370880.00 
2020-07-27 14:30:40,705:INFO: - New best score (-197160514126517370880.00)!
2020-07-27 14:30:40,705:INFO: - Saving model...
2020-07-27 14:30:41,301:INFO: - Saved model in results/small/model_weights/
2020-07-27 14:30:41,301:INFO: - Elapsed time: 5.06, learning rate: 0.00014

求指点，谢谢
